Assets/UIElements/_Elements/XRUI_Feedback/Scripts/XRUI_Feedback.cs

```diff
@@ -145,22 +149,30 @@ private void CreateNearCollider()
                 throw new Exception("Unknow NearColliderType selected");
         }
         nearCollider.isTrigger = true;
-
         xrNearDetection = nearColliderObject.AddComponent<XRUI_NearDetectionLocked.XRUI_NearDetection>();
+
+        if (allowDistanceEvents)
+        {
+            XRBaseInteractable interactable = gameObject.GetComponent<XRBaseInteractable>();
+            if (interactable == null)
+                interactable = gameObject.GetComponentInChildren<XRBaseInteractable>();
+
+            if (interactable == null)
+                interactable = gameObject.AddComponent<XRSimpleInteractable>();
+
+            if (interactable != null)
+            {
+                interactable.onHoverEnter.AddListener((XRBaseInteractor) => { xrNearDetection.OnEnterAction(XRBaseInteractor); });
+                interactable.onHoverExit.AddListener((XRBaseInteractor) => { xrNearDetection.OnExitAction(XRBaseInteractor); });
+            }
+        }
```

Comment:
Same as other comments regarding deal with this on BaseClass

Reply:
Changed